src/web/core/src/main/java/org/geoserver/web/CapabilitiesHomePagePanel.java

```diff
@@ -85,10 +88,20 @@ public CapabilitiesHomePagePanel(final String id, final List<CapsInfo> capsLinks
 
         super(id);
 
+        // Group CapInfo by service id
         final Map<String, List<CapsInfo>> byService = new HashMap<>();
         for (CapsInfo c : capsLinks) {
             final String key =
                     c.getService().toLowerCase(); // to avoid problems with uppercase definitions
+            if (key.equals("wms")
```

Comment:
This feels a bit brittle. Could we ask the `ServicesPanel` what `key`s it handles  or if a `key` is handled instead?

Reply:
I think I can remove these now that the ``CapabilitiesHomePageLinkProvider`` instances have been removed CapabilitiesHomePageLinkProvider.

This is only still present in the codebase for any community modules

Reply:
It is more what keys ``ServiceInfoCapabilitiesProvider`` handles; but perhaps I can ask that extension point.

Reply:
Okay I was able to process a good skip list.